### PR TITLE
bugfix/#5734-Change-deactivate-pop-up-according-to-mock-up

### PR DESCRIPTION
**Pull request by NadiiaStoiko**
Before:
The new pop-up window appeared with all selected points as text in 1 language and only the button 'Скасувати' and such text is visible in the upper right corner of the popup window: 'Введіть наступні дані АНГЛІЙСЬКОЮ мовою".
After:
The new pop-up window named “Підтвердити деактивацію” appeared with all selected points as text in 2 languages and the buttons 'Деактивувати' and 'Скасувати'.

### CODE REVIEW

File: src/app/ubs/ubs-admin/components/shared/components/tariff-deactivate-confirmation-pop-up/tariff-deactivate-confirmation-pop-up.component.html [hunk 1]
```diff
@@ -32,21 +32,31 @@
...
+        <label class="label">{{ getLangValue(regionLabelEn, regionLabelUa) }}</label>
+        <p class="text">{{ getLangValue(regionEnglishName, regionName) }}</p>
       </div>
     </div>
     <div class="form-block">
       <div class="ua-part">
         <label class="label">{{ 'ubs-tariffs.city' | translate }}</label>
-        <p *ngFor="let locationName of locationNames" class="text">{{ locationName }}</p>
+        <div *ngIf="lang">
+          <p *ngFor="let locationName of locationNames" class="text">{{ locationName }}</p>
```
Comment (atsa21) [inline src/app/ubs/ubs-admin/components/shared/components/tariff-deactivate-confirmation-pop-up/tariff-deactivate-confirmation-pop-up.component.html]: You can add another function:
getLangArrayValue(uaValue: string[], enValue[]) {
return this.languageService.getLangValue(uaValue, enValue) as string[];
}
replace locationNames with getLangArrayValue(locationNames, locationEnglishNames) and remove *ngIf

File: src/app/ubs/ubs-admin/components/shared/components/tariff-deactivate-confirmation-pop-up/tariff-deactivate-confirmation-pop-up.component.ts [hunk 2]
```diff
@@ -26,6 +33,13 @@ export class TariffDeactivateConfirmationPopUpComponent implements OnInit {
   isRestore: boolean;
   isDeactivate: boolean;
   isDeactivatePopup = true;
+  lang = false;
```
Comment (atsa21) [inline src/app/ubs/ubs-admin/components/shared/components/tariff-deactivate-confirmation-pop-up/tariff-deactivate-confirmation-pop-up.component.ts]: After added new function delete this.lang variable

File: src/app/ubs/ubs-admin/components/shared/components/tariff-deactivate-confirmation-pop-up/tariff-deactivate-confirmation-pop-up.component.html (hunk 1 above)
Comment (NadiiaStoiko) [reply src/app/ubs/ubs-admin/components/shared/components/tariff-deactivate-confirmation-pop-up/tariff-deactivate-confirmation-pop-up.component.html]: Thank you! Change code according to recommendations

File: src/app/ubs/ubs-admin/components/shared/components/tariff-deactivate-confirmation-pop-up/tariff-deactivate-confirmation-pop-up.component.ts (hunk 2 above)
Comment (NadiiaStoiko) [reply src/app/ubs/ubs-admin/components/shared/components/tariff-deactivate-confirmation-pop-up/tariff-deactivate-confirmation-pop-up.component.ts]: Thank you! Change code according to recommendations

**Review by AlexW8719 — APPROVED** [review]

**Review by AlexW8719 — APPROVED** [review]

**Review by katia9261 — APPROVED** [review]

**Review by atsa21 — APPROVED** [review]

**Review by ruslan-teh — APPROVED** [review]

**Review by hnativlyubomyr — APPROVED** [review]

**Pull request merged**